.gitmodules

```
@@ -26,3 +26,6 @@
 [submodule "components/core/submodules/outcome"]
 	path = components/core/submodules/outcome
 	url = https://github.com/ned14/outcome.git
+[submodule "components/log-viewer-webui/yscope-log-viewer"]
+    path = components/log-viewer-webui/yscope-log-viewer
+    url = git@github.com:y-scope/yscope-log-viewer.git
```

```suggestion
[submodule "components/log-viewer-webui/yscope-log-viewer"]
	path = components/log-viewer-webui/yscope-log-viewer
	url = https://github.com/y-scope/yscope-log-viewer.git
```
1. Let's use the https git repo link.
2. Also we need to add the `clp/components/log-viewer-webui` reference to VCS.